moveit_planners/trajopt/CMakeLists.txt

```diff
@@ -22,6 +12,7 @@ find_package(catkin REQUIRED
       rosparam_shortcuts
       trajopt
 )
+moveit_build_options()
```

Comment:
Please remove this `moveit_build_options()` calls. In not-yet ported packages like this one we would enable `moveit_package()`, anyway.